images/runtime/node/Dockerfile.template

```diff
@@ -22,6 +22,13 @@ ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
 RUN mkdir -p /node_modules \
  && chmod 777 /node_modules
 
+RUN set -e \
```

Comment:
I did not bother to have this piece of code for older versions of Node runtime images since they are not using `6` version of npm.